packages/beacon-node/src/chain/produceBlock/computeNewStateRoot.ts

```diff
@@ -41,5 +41,8 @@ export function computeNewStateRoot(
     metrics
   );
 
-  return postState.hashTreeRoot();
+  const {attestations, syncAggregate, slashing} = postState.rewards;
+  const proposerReward = BigInt(attestations + syncAggregate + slashing);
```

Comment:
> ~should be bigint from the beginning itself unless they are in "increments" because there could be loss of precision at time of setting or adding~
> since all balances are in GWEI, i guess this is ok

I am debating whether to return `proposerReward` as BigInt or number.
2^53 - 1 GWei is like 9million Eth, so there is no point of having them as BigInt unless we want to be semantic in the code to have type GWei aka UintBn64 aka BigInt

Reply:
proposerReward as bigint is fine